### PR TITLE
document the configuration for topology manager

### PR DESCRIPTION
**Pull request by fromanirh**
**Which issue(s) this PR fixes**:
Related to https://github.com/kubernetes/kubernetes/issues/83481

### CODE REVIEW

File: contributors/devel/sig-node/e2e-node-tests.md [hunk 1]
```diff
@@ -251,3 +251,43 @@ The PR builder runs tests against the images listed in [jenkins-pull.properties]
...
+
+Some topology manager tests require minimal knowledge of the host topology in order to be performed.
+The required information is to which NUMA node in the system are the SRIOV device attached to.
+The test code tries to autodetect the information it needs, skipping the relevant tests if the autodetection fails.
+
+You can override the autodetection adding annotations to the the config map like this example:
+```yaml
+metadata:
+  annotations:
+    pcidevice_node0: 1
```
Comment (vpickard) [inline contributors/devel/sig-node/e2e-node-tests.md]: Should this be:
pcidevice_node0: "1"

and so on.... I had to add quotes like this to get it to parse on my system.

Comment (fromanirh) [reply contributors/devel/sig-node/e2e-node-tests.md]: Right! fixing

**Comment by fromanirh** [comment]
ping? who can approve this PR?

**Pull request merged**